cmd/exo/logs.go

```diff
@@ -109,6 +113,22 @@ func tailLogs(ctx context.Context, workspace api.Workspace, logRefs []string) er
 			fmt.Printf("%s %s\n", prefix, event.Message)
 		}
 		in.Cursor = &output.NextCursor
+
+		if stopOnError {
+			descriptions, err = workspace.DescribeProcesses(ctx, &api.DescribeProcessesInput{})
+			if err != nil {
+				return fmt.Errorf("failed to check status of processes: %w", err)
+			}
+
+			for _, proc := range descriptions.Processes {
+				for _, id := range logIDs {
+					if proc.ID == id && !proc.Running {
+						return fmt.Errorf("process stopped running")
```

Comment:
nit: use `errors.New` when there are no dynamic formatting bits in the error message. Though, here, I think it would make sense to say _which_ process stopped running. Ideally, the component name. Something like:

```suggestion
						return fmt.Errorf("process stopped running: %q", proc.Name)
```